stl/A_kuafu/Triplewo/src/00-hello_world/main.cpp

```diff
@@ -0,0 +1,10 @@
+#include<iostream>
+#include <string>
+using namespace std;
```

Comment:
You may not use a using-directive to make all names from a namespace available.

see: google style.https://google.github.io/styleguide/cppguide.html#Namespaces